scoreCard.js

```diff
@@ -0,0 +1,54 @@
+class ScoreCard {
+  constructor() {
+    this.scores = [];
+    this.totalScore = 0;
+  }
+
+  frameResult(array) {
+    this.scores.push(array);
+  };
+
+  printFrameScores() {
+    this.scores.map((score, index) => {
+      console.log(`Frame: ${index + 1}, First roll: ${score[0]}, Second roll: ${score[1]}, Bonus: ${score[2]}`);
```

Comment:
It's great how you print this out so you can see the rolls and bonuses for each frame! My one suggested improvement would be that the final frame is listed as one frame with all three rolls, frame 10 rather than frame 10,11,12. But that's probably my own interpretation of the bowling scorecard instructions/scoring system and maybe down to personal preference!